.teamcity/subprojects.json

```diff
@@ -850,7 +850,7 @@
     "name": "smoke-ide-test",
     "path": "subprojects/smoke-ide-test",
     "unitTests": false,
-    "functionalTests": false,
+    "functionalTests": true,
```

Comment:
@blindpirate Right, this is not intended. We want it to run like a smoke tests.
Also, this PR is changing source set from `smokeIdeTest` to a regular `integTest`. Will it lead to unintentional run `smoke-ide-tests` during other check, that involving running of all `integTests`?

Reply:
@blindpirate We want to avoid this. Is there other way apart from using another sourceSet? But if it's a most recommended way, we could revert that sourceSet changes back

Reply:
@blindpirate Yes, I can try it, but at the same time it will brings some implicit behavior: 
When you see `integTest` source set, it's implies that it will run with other functional tests. And if I'll make `"functionalTests": false` it can be confusing a bit. 

From other side, having a dedicated source set (as it was before) is explicitly tells that there is a logic to run it accordingly.

WDYT? cc @bamboo

Reply:
@blindpirate Huh, when I'm making `"functionalTests": false` (unchanged essentially), then `:checkSubprojectsInfo` is failing:

```
:checkSubprojectsInfo'. org.gradle.api.GradleException: New project(s) added without updating subproject JSON. Please run `:generateSubprojectsInfo`
```

`:generateSubprojectsInfo`, in it's turn, is generating `"functionalTests": true` for `smoke-ide-test`, what is breaking [this](https://github.com/gradle/gradle/blob/3cef92b48da12c3e4d6a12f8bab18c54138f21fb/build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/CheckSubprojectsInfo.kt#L29)  condition.

So apparently, it would be correct to revert source set changes